spec/factories/payments.rb

```diff
@@ -5,8 +5,14 @@
     end
 
     claims do
+      claim_attributes = {
```

Comment:
So we're setting these explicitly so that we can ensure the claims share the same personal details. Is there a better, more expressive name for this variable to help the reader understand that?

Reply:
:+1: changed it to `personal_details`.